core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -52,6 +52,16 @@ private[spark] class CoarseGrainedExecutorBackend(
     driver ! RegisterExecutor(executorId, hostPort, cores)
     context.system.eventStream.subscribe(self, classOf[RemotingLifecycleEvent])
   }
+  
+  override def isValidMessage(msg: Any) = msg match {
+    case disEvt: DisassociatedEvent =>
+      if (disEvt.remoteAddress == driver.anchorPath.address) {
```

Comment:
Maybe I'm not understanding this correctly. I thought whether a DisassociatedEvent is valid is universal to all actors used in Spark.  Is it not?